src/presentation/ui/presenter/databasePresenter.go

```diff
@@ -0,0 +1,71 @@
+package presenter
+
+import (
+	"log/slog"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"github.com/speedianet/os/src/domain/entity"
+	"github.com/speedianet/os/src/domain/valueObject"
+	internalDbInfra "github.com/speedianet/os/src/infra/internalDatabase"
+	"github.com/speedianet/os/src/presentation/service"
+	uiHelper "github.com/speedianet/os/src/presentation/ui/helper"
+	"github.com/speedianet/os/src/presentation/ui/page"
+	presenterDto "github.com/speedianet/os/src/presentation/ui/presenter/dto"
+)
+
+type DatabasesPresenter struct {
+	databaseService *service.DatabaseService
+}
+
+func NewDatabasesPresenter(
+	persistentDbSvc *internalDbInfra.PersistentDatabaseService,
+) *DatabasesPresenter {
+	return &DatabasesPresenter{
+		databaseService: service.NewDatabaseService(persistentDbSvc),
+	}
+}
+
+func (presenter *DatabasesPresenter) getSelectedDatabaseTypeSummary(
+	rawDatabaseType string,
+) (selectedDbTypeSummary presenterDto.SelectedDatabaseTypeSummary, err error) {
+	databaseType, err := valueObject.NewDatabaseType(rawDatabaseType)
+	if err != nil {
+		return selectedDbTypeSummary, err
+	}
+
+	isInstalled := false
+
+	requestInput := map[string]interface{}{"dbType": databaseType.String()}
```

Comment:
I think we named this requestBody on all the other controllers.

Reply:
Done!